src/main/java/uk/gov/pay/directdebit/notifications/services/UserNotificationService.java

```diff
@@ -93,4 +113,9 @@ private static String formatToPounds(long amountInPence) {
         return BigDecimal.valueOf(amountInPence, 2).toString();
     }
 
+    private void logMissingSunName(EmailTemplate template, Mandate mandate) {
+        LOGGER.error("Mandate {} does not have a Service User Number. " +
+                "Email with template {} is not being sent.", mandate, template);
```

Comment:
`Mandate` doesn’t have a nice `toString()` method. Probably want to do grab its external ID and reference and include them in the message.